src/inspirehep-holdingpen-js/holdingpen/holdingpen.directives.js

```diff
@@ -270,6 +270,11 @@
             $window.location = url;
           },
 
+          saveAndRedirect: function(url) {
+            HoldingPenRecordService.updateRecord($scope.vm, $scope.workflowId);
```

Comment:
Is this call synchronous? Otherwise it should be something like `HoldingPenRecordService.updateRecord($scope.vm, $scope.workflowId).then(function(){})`

Reply:
Updated the code